src/components/dialogs/parameters/widget/parameter-float.tsx

```diff
@@ -14,24 +14,35 @@ export interface ParameterFloatProps {
     label: string;
     style: any;
     adornment: any;
+    tooltip?: string;
 }
 
 export const ParameterFloat = ({
     name,
     label,
     style,
     adornment,
+    tooltip,
 }: ParameterFloatProps) => {
     return (
         <>
-            <Grid item container direction={'row'} spacing={1} paddingTop={3}>
```

Comment:
previous grid was an item, it means the parent (parent of this component) was a grid container. It should not be ok to replace it with the Tooltip component?

Reply:
Done